modules/apps/asset/asset-list-service/service.xml

```diff
@@ -145,9 +145,13 @@
 		<!-- Other fields -->
 
 		<column name="assetListEntryId" type="long" />
+		<column name="priority" type="int" />
 		<column name="segmentsEntryId" type="long" />
 		<column name="typeSettings" type="String" />
 		<column name="lastPublishDate" type="Date" />
+		<order by="asc">
```

Comment:
This has to go in its own section (see example: https://github.com/liferay/liferay-portal/blob/304c2a1931bc1c673a5da993a1051217ffbaa987/modules/apps/segments/segments-service/service.xml#L36-L40).

Reply:
Added proper format to the order by clause, adding a comment and new empty lines